lib/note-list/index.tsx

```diff
@@ -212,6 +213,19 @@ export class NoteList extends Component<Props> {
     this.toggleShortcuts(false);
   }
 
+  computedLabel() {
+    switch (this.props.collection.type) {
+      case 'tag':
+        return 'Notes With Selected Tag';
+      case 'trash':
+        return 'Trash';
+      case 'untagged':
+        return 'Untagged Notes';
+      default:
+        return 'All Notes';
+    }
+  }
```

Comment:
we don't need to refer to the class instance. can we move this into the module-global scope?

```ts
const sidebarTitle = (collection: Collection) => {
	switch (collection.type) {
		…
	}
}
```

on this I raise two questions:
 - should we extract this function to another file where this module and the list item module can use it together?
 - should we be interpolating the selected tag name here or leave it at "selected tag"?

Reply:
Moving it from a class method to the module scope makes sense. ✅ 

> should we extract this function to another file where this module and the list item module can use it together?

I originally avoided doing this because (1) it is only used in two places currently and (2) I hope its [usage here eventually goes away](). That said, I'm happy to abstract it if you'd like. What do you think?

> should we be interpolating the selected tag name here or leave it at "selected tag"?

Yes, it may be a improvement for the title to state the tag name rather than a generic statement. One caveat is that that tag is likely to be visually truncated. Would you like me to change the visual title and ARIA label to be the tag value itself?

<img width="302" alt="Screen Shot 2021-03-09 at 09 05 25" src="https://user-images.githubusercontent.com/438664/110491143-98bf2900-80b6-11eb-8552-c24921ea0954.png">

Reply:
>I think we used to show the tag name without the additional titling, which didn't have this problem.

Indeed. I confirmed that. That fact is also related to https://github.com/Automattic/simplenote-electron/pull/2726#discussion_r589807280.

>this is beyond me at the moment. feel free to reach out to others or make a call for what you think is best.

Alright. Hopefully someone else will also find time to review and provide feedback.

Reply:
@SylvesterWilmott what say you?

Reply:
The title in the title bar when a tag is selected should be `tag name` to be consistent with `all notes` or `trash`.

Regarding the search placeholder, electron is the only remaining platform where search is not global and so we have this need to imply search is contained only within the selected tag. Once search becomes global (searches all notes all the time) then the search placeholder will read "Search notes and tags". Until then we can try "Search `tag name`" or "Search in `tag name`" to avoid that truncation that looks like it would be a fairly common occurance.

Reply:
Thanks for the feedback. I updated the search placeholder to be `Search in <tag_name>`. I also applied an ellipsis when the placeholder overflows the input width.

<img width="383" alt="image" src="https://user-images.githubusercontent.com/438664/110822498-bd520700-8256-11eb-8b4b-da01df93a427.png">